lmdeploy/serve/turbomind/deploy.py

```diff
@@ -521,7 +725,9 @@ def main(model_name: str,
          model_format: str = 'hf',
          tokenizer_path: str = None,
          dst_path: str = './workspace',
-         tp: int = 1):
+         tp: int = 1,
+         quant_path: str = None,
+         group_size: int = 0):
```

Comment:
How about setting the default value of `group_size` 128?

Reply:
It's confusing to set `group_size` to a non-trivial value when W4 is not used.